var/spack/repos/builtin/packages/qmd-progress/package.py

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Copyright (c) 2017, Los Alamos National Security, LLC
+# Produced at the Los Alamos National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class QmdProgress(CMakePackage):
+    """PROGRESS: Parallel, Rapid O(N) and Graph-based Recursive Electronic
+    Structure Solver. LA-CC-16-068
+    This library is focused on the development of general solvers that are
+    commonly used in quantum chemistry packages."""
+
+    homepage = "https://github.com/lanl/qmd-progress"
+    url      = "https://github.com/lanl/qmd-progress"
+
+    version('develop', git='https://github.com/lanl/qmd-progress', branch='master')
+    version('1.0.0', git='https://github.com/lanl/qmd-progress', tag='v1.0.0')
+
+    variant('graphlib', default=False, description='Build with Metis Suppport')
+    variant('mpi', default=True, description='Build with MPI Support')
+
+    depends_on('cmake', type='build')
+
+    depends_on("bml")
+    depends_on('mpi', when="+mpi")
+    depends_on('metis', when="+graphlib")
```

Comment:
Might want to stick to single quotes or double quotes instead of mixing them. I usually prefer single quotes unless the string contains an apostrophe.